htdocs/comm/propal/card.php

```diff
@@ -1998,7 +1998,27 @@
 	}
 
 	// Other attributes
-	include DOL_DOCUMENT_ROOT.'/core/tpl/extrafields_add.tpl.php';
```

Comment:
You remove the geneic code with specific one ?
We try currently to go on the oposite direction...